sw-precache.js

```diff
@@ -119,11 +120,11 @@ module.exports = function(params) {
   params.logger(util.format('Total precache size is about %s for %d resources.',
     formatBytesAsString(cumulativeSize), relativeUrls.length));
 
-  var templateBuffer = fs.readFileSync(params.templateFilePath);
-  return _.template(templateBuffer, {
-    handleFetch: params.handleFetch,
-    importScripts: params.importScripts ? params.importScripts.map(JSON.stringify).join(',') : null,
-    includeCachePolyfill: params.includeCachePolyfill,
-    precacheConfig: JSON.stringify(precacheConfig)
-  });
+  return fs.createReadStream(params.templateFilePath)
+    .pipe(lodashTemplateStream({
+      handleFetch: params.handleFetch,
+      importScripts: params.importScripts ? params.importScripts.map(JSON.stringify).join(',') : null,
+      includeCachePolyfill: params.includeCachePolyfill,
+      precacheConfig: JSON.stringify(precacheConfig)
+    }));
```

Comment:
Go with `fs.readFile` instead.

The module you're using here is not working correctly at all.